src/modules/fields/index.js

```diff
@@ -6,8 +6,8 @@ import {
 } from './baseFields.js'
 import { extendFields } from './nestedFields.js'
 
-export const getFieldsStringByType = (type) =>
-    getAllFieldObjectsByType(type)
+export const getFieldsStringByType = (type, withSubscribers) =>
```

Comment:
Minor, I would use an object for the "options" here:
```suggestion
export const getFieldsStringByType = (type, { withSubscribers }) =>
```